ext/js/language/ja/japanese-transforms.js

```diff
@@ -708,8 +926,18 @@ export const japaneseTransforms = {
                 suffixInflection('來', '來る', [], ['vk']),
             ],
         },
-        'negative': {
-            name: 'negative',
+        '-nai': {
```

Comment:
-masen is also in this group, so a bit strange when it's named '-nai'. Could move -masen to its own group or keep this named 'negative'

Reply:
I'm fine with reverting back to `negative`. To be more accurate, ません should be under `-n`, because ん (or originally ぬ) is attached to the mizenkei of masu, ませ. But then this would probably cause more confusion 😕